pkg/cmd/hgctl/dashboard.go

```diff
@@ -81,6 +88,8 @@ func newDashboardCmd() *cobra.Command {
 			"Default is true which means hgctl dashboard will always open a browser to view the dashboard.")
 	dashboardCmd.PersistentFlags().StringVarP(&addonNamespace, "namespace", "n", "higress-system",
 		"Namespace where the addon is running, if not specified, higress-system would be used")
+	dashboardCmd.PersistentFlags().StringVarP(&bindAddress, "listen", "l", "localhost", "The address to bind to")
+	dashboardCmd.PersistentFlags().BoolVar(&docker, "docker", false, "Search higress console from docker")
```

Comment:
the docker flag is now only used by console subcommand, it would be better to move it under console subcommand